url_utils/icon_url_utils/connection/connection.py

```diff
@@ -0,0 +1,10 @@
+import insightconnect_plugin_runtime
+from .schema import ConnectionSchema
+
+
+class Connection(insightconnect_plugin_runtime.Connection):
+    def __init__(self):
+        super(self.__class__, self).__init__(input=ConnectionSchema())
+
+    def connect(self, params):
+        pass
```

Comment:
Needs a `test` function for a connection test. There's no connection, but it needs a function anyway. It should always succeed.

```
def test(self):
    return True
```
IIRC

Reply:
I haven't seen this in a few random plugins I checked, but I did add it in anyway.